dashboard/final-example/auth.config.ts

```diff
@@ -11,8 +11,12 @@ export const authConfig = {
   callbacks: {
     authorized({ auth, request: { nextUrl } }) {
       const isLoggedIn = !!auth?.user;
-      const isOnDashboard = nextUrl.pathname.startsWith('/dashboard');
-      if (isOnDashboard) {
+      const protectedPaths = ['/dashboard', '/customers', '/invoices'];
+      const isProtectedPath = protectedPaths.some((path) =>
+        nextUrl.pathname.startsWith(path),
+      );
```

Comment:
Hey @tylerhoward15, thank you for the contribution. This was resolved in this PR: https://github.com/vercel/next-learn/pull/355

Are you still seeing broken images? I'm not able to reproduce. 

In either case: 
- The protected pathname starts with /dashboard. E.g. `/dashboard/customers`. So `startsWith` would evaluate to `false`.
- I think it's slightly better practise to protect a route and its children, so that if we add or change the name of a child segment, they'd automatically be protected.